lib/spoom/sorbet/sigils.rb

```diff
@@ -0,0 +1,39 @@
+# typed: true
+# frozen_string_literal: true
+
+module Spoom
+  module Sorbet
+    module Sigils
+      extend T::Sig
+
+      VALID_STRICTNESS = ["ignore", "false", "true", "strict", "strong"].freeze
+      SIGIL_REGEXP = /^#\s*typed\s*:\s*(\w*)$/.freeze
+
+      # returns the full sigil comment string for the passed strictness
+      sig { params(strictness: String).returns(String) }
+      def self.sigil(strictness)
```

Comment:
```suggestion
      def self.sigil_string(strictness)
```

feels a bit more self-explanatory